README.md

```diff
@@ -6,11 +6,15 @@ This is a stub implementation of the EIS Integration Framework backend for the B
 
 Get BS Periods
 
-    'https://breathing-space-if-stub.protected.mdtp/breathing-space-stateless/api/v1/BS000005A/periods' --header 'CorrelationId: 75e4cdc2-eecb-11ea-913a-0a9f09f70a70' --header 'OriginatorId: DS2_BS_UNATTENDED' --header 'UserId: 0000000'
+    'https://breathing-space-if-stub.protected.mdtp/individuals/breathing-space-stateless/NINO/BS000005A/periods' --header 'CorrelationId: 75e4cdc2-eecb-11ea-913a-0a9f09f70a70' --header 'OriginatorId: DS2_BS_UNATTENDED' --header 'UserId: 0000000'
```

Comment:
Maybe it's worth to remove the host from the README. WDYT? This is something that we have in our config files which I think that maybe we should not expose.

Reply:
Ok, done.